test/e2e/prometheus_test.go

```diff
@@ -167,3 +167,35 @@ func TestPrometheusAlertmanagerAntiAffinity(t *testing.T) {
 		t.Fatal("Can not find pods: prometheus-k8s or alertmanager-main")
 	}
 }
+
+func TestPrometheusTerminationMessagePolicy(t *testing.T) {
+	pods, err := f.KubeClient.CoreV1().Pods(f.Ns).List(
+		metav1.ListOptions{FieldSelector: "status.phase=Running"})
```

Comment:
Can't we just select Pod Name here to be "cluster-monitoring-operator", that way we don't need the if condition for namespace and pod name on line 185.

Reply:
Thanks for the idea. Let me investigate that path please.

Reply:
I have no idea have how to do that. Try different cases without result. The problem is that cluster operator name  joined with randomly generated suffix and I had could not get it without knowing this random part.

Reply:
But If you know something, you are welcome to ping me )

Reply:
/assign @LiliC

Reply:
Since it should always be just one pod, you can just use a `.Get` as well.